shared-libs/registration-utils/package-lock.json

```diff
@@ -1,1181 +1,5 @@
 {
   "name": "@medic/registration-utils",
   "version": "1.1.1",
```

Comment:
Same here, is this package-lock okay? lots of red

Reply:
probably not!  I'll revert this and subsequent "is this ok?" below. This one fixed in 21769adfa